### DISCUSSION TITLE
PS Vita Custom Firmware on the Steam Deck

### DISCUSSION THREAD

**Discussion by DrinkingSubset** [xerpi/vita2hos#27] [Ideas]
It would be really cool if we could run the PlayStation Vita custom firmware on the steam deck. It would be very nice if we achieve this so that we can also get adrenaline Native PSP emulator running through PlayStation Vita custom firmware natively. 

I would guess for some help follow or contact @TheOfficalFloW here on github for some help and @PPSSPP_emu over on Twitter. To see if you guys would collaborate in bringing this to life to the Steam Deck. I'm very excited what you guys think about this!

**Comment by TSRBerry** [comment]
How is that related to vita2hos?

**Comment by ghost** [comment]
Adrenaline wouldn't run on Deck. Adrenaline relies on the internal PSP parts of the Vita.

Additionally, the Steam Deck has capable PSP emulation as far as I'm aware?

>

**Reply by DrinkingSubset** [reply]
That's why I said A PS Vita custom firmware emulated on the steam deck would be nice to have to run adrenaline you cannot run plug-ins through PPSSPP on the steam deck.

**Comment by ghost** [comment]
That's impossible to do. The Deck doesn't have the necessary hardware components to make Adrenaline work--it doesn't have a built-in PSP, unlike the Vita.

>

**Reply by DrinkingSubset** [reply]
![image](https://user-images.githubusercontent.com/80879977/183528692-7fc3eaaa-21a2-4f07-8581-8e0962d6b773.jpeg)

**Comment by Masamune3210** [comment]
Adrenaline can't and probably will never work on anything that isn't a Vita, it relies on the PSP HAL built into the Vita firmware and requires actual PSP hardware that is on the Vita motherboard. You want plugin support in PPSSPP, go bother them, it has nothing to do with this project lol

**Comment by TSRBerry** [comment]
Sorry, but this discussion doesn't seem to be related to vita2hos at all, so I'm going to lock this.
